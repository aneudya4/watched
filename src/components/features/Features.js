import React from 'react';
import './features.css';

const Features = () => (
  <section className="features">
    <div className="all-features">
      <h3>
        Features
        <span> . </span>
      </h3>
      <div className="features-container">
        <div className="feature">
          <span className="icon">
            <i className="fas fa-gift" />
          </span>
          <div className="features-details">
            <h4>Great Entertaiment Library</h4>
            <p>Our library includes tons of information about movies.</p>
          </div>
        </div>
        <div className="feature">
          <span className="icon">
            <i className="fas fa-list" />
          </span>
          <div className="features-details">
            <h4>Keep a record of your media</h4>
            <p>Combine the best content into one easy-to-use web app.</p>
          </div>
        </div>
      </div>
    </div>
  </section>
);

export default Features;
