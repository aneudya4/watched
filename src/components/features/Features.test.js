/* eslint-disable no-undef */
// /* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Features from './Features';

describe('Test on the <Features/> Component', () => {
  it('renders App without crashing', () => {
    const wrapper = shallow(<Features />);

    expect(wrapper).toMatchSnapshot();
  });
});
