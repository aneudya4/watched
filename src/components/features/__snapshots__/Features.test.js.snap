// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on the <Dashboard/> Component renders App without crashing 1`] = `
<section
  className="features"
>
  <div
    className="all-features"
  >
    <h3>
      Features
      <span>
         . 
      </span>
    </h3>
    <div
      className="features-container"
    >
      <div
        className="feature"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-gift"
          />
        </span>
        <div
          className="features-details"
        >
          <h4>
            Great Entertaiment Library
          </h4>
          <p>
            Our library includes tons of information about movies.
          </p>
        </div>
      </div>
      <div
        className="feature"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-list"
          />
        </span>
        <div
          className="features-details"
        >
          <h4>
            Keep a record of your media
          </h4>
          <p>
            Combine the best content into one easy-to-use web app.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;

exports[`Test on the <Features/> Component renders App without crashing 1`] = `
<section
  className="features"
>
  <div
    className="all-features"
  >
    <h3>
      Features
      <span>
         . 
      </span>
    </h3>
    <div
      className="features-container"
    >
      <div
        className="feature"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-gift"
          />
        </span>
        <div
          className="features-details"
        >
          <h4>
            Great Entertaiment Library
          </h4>
          <p>
            Our library includes tons of information about movies.
          </p>
        </div>
      </div>
      <div
        className="feature"
      >
        <span
          className="icon"
        >
          <i
            className="fas fa-list"
          />
        </span>
        <div
          className="features-details"
        >
          <h4>
            Keep a record of your media
          </h4>
          <p>
            Combine the best content into one easy-to-use web app.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
